config/locales/notices/en.yml

```diff
@@ -26,6 +26,9 @@ en:
         access government services. We use industry-standard monitoring and
         security practices to protect your data. %{link}
       link: Learn more ›
+    use_diff_email:
+      text: Or, %{link}
```

Comment:
I feel good about that! Will stamp. Also a good topic for today's process meeting